src/js/include/queue-manager.js

```diff
@@ -0,0 +1,84 @@
+import { updateResultFailure, updateResult } from "./render";
+import $ from "jquery";
+
+export function initQueue(itemsToScan, activeOnly) {
+  // Reset the queue.
+  window.phpcompat.queue = [];
+
+  itemsToScan.plugins.forEach((plugin) => {
+    if ("yes" === plugin.active || "no" === activeOnly) {
+      window.phpcompat.queue.push({
+        ...plugin,
+        type: "plugin",
+      });
+    }
+  });
+
+  itemsToScan.themes.forEach((theme) => {
+    if ("yes" === theme.active || "no" === activeOnly) {
+      window.phpcompat.queue.push({
+        ...theme,
+        type: "theme",
+      });
+    }
+  });
+}
+
+export function executeJob(job, cb) {
+  // TODO indicate current plugin/theme in progress
```

Comment:
Is this `TODO` going to be handled in this PR or separately?

Reply:
This TODO was already resolved before with the spinner image. Removing it in https://github.com/10up/phpcompat/pull/3/commits/99bb0468259a0b2b5efc5f42d9a17fb15d806fe5